scss/bitstyles/objects/button--dropdown/settings.scss

```diff
@@ -0,0 +1,40 @@
+//
+// Base styles ////////////////////////////////////////
+
+$bitstyles-button-dropdown-transition:
+  color $bitstyles-transition-duration $bitstyles-transition-easing,
+  background-color $bitstyles-transition-duration $bitstyles-transition-easing,
+  border $bitstyles-transition-duration $bitstyles-transition-easing,
+  box-shadow $bitstyles-transition-duration $bitstyles-transition-easing !default;
+
+//
+// Base colors ////////////////////////////////////////
+
+$bitstyles-button-dropdown-color: palette('gray', '60') !default;
```

Comment:
Wow TIL `!default` 💡 

> You can assign to variables if they aren’t already assigned by adding the !default flag to the end of the value.

Reply:
Yup :D It’s an interesting kinda-backwards override, for when you can’t intercept the setting of a variable (the settings file is imported in the component file, there’s no way to stop that when you’re a user of the npm package)